modules/system/assets/css/styles.css

```diff
@@ -368,3176 +368,7744 @@ blockquote.pull-right .small:after{content:'\00A0 \2014'}
 blockquote:before,
 blockquote:after{content:""}
 address{margin-bottom:20px;font-style:normal;line-height:1.42857143}
+.close{float:right;font-size:21px;font-weight:bold;line-height:1;color:#000;text-shadow:0 1px 0 #fff;font-family:sans-serif;opacity:0.2;filter:alpha(opacity=20)}
+.close:hover,
+.close:focus{color:#000;text-decoration:none;cursor:pointer;opacity:0.5;filter:alpha(opacity=50)}
+button.close{padding:0;cursor:pointer;background:transparent;border:0;-webkit-appearance:none}
+.icon-0::before,
+.wn-icon-0::before,
+.oc-icon-0::before{content:"\30"}
```

Comment:
Can we break these icon content definitions into their own compiled file and just reference them as needed so they aren't adding 11k lines to several different asset files? Also, can you verify that the oc- BC prefix is only applied on the old icon classes?

Reply:
I'm not sure I follow. Do you mean separating the different icon types? We haven't got a mechanism yet that detects when icons are in use, so we can't conditionally load the icon styles. I just followed the same compilation process that the original icons used.

Reply:
As for the original oc- polyfills, it's only shimming the original LESS variables, but it is applying an oc prefix to all icons, including the new ones. I can limit this to just the original icons if needed.

Reply:
@bennothommo I mean that the styles.css and storm.css both had ~10k lines added / changed, would probably be better to load those styles in an icons.css that gets added via the AssetMaker trait methods whenever the its required. If it's not too much bother I'd prefer if the oc- prefixes were only added to the original icon names, that's a lot of duplicated code that is unnecessary and shouldn't be encouraged to be used.

Reply:
I think it's reasonable to assume that the icons are used *everywhere* in the Backend. Plus, there would be people who are using icons in an ad-hoc way (myself included) who would be forced to add an `addCss` call to their plugins.

My thoughts are this: for 50kb-60kb extra, you're getting nearly 1100 extra icons. It wouldn't be a tall order for a lot of people.

Reply:
How about at least we provide it via a CSS import at the top of the file then? Then it's being cached separately and can be shared between the two compiled files.

Reply:
I've opted to compile the icons into a separate icon CSS file, which is loaded on all Backend pages. I've re-adjusted how the icons are compiled as well so that the base icon styles can be loaded, and allow us to be selective on icons to load, like the exception pages and maintenance or missing database views. This was previously loading all icons (through `modules/system/assets/css/style.css`).